Лабораторная работа 2/task3.5.3.py

```diff
@@ -0,0 +1,65 @@
+# TODO  Напишите функцию count_letters
+
+
+main_str = """
+У лукоморья дуб зелёный;
+Златая цепь на дубе том:
+И днём и ночью кот учёный
+Всё ходит по цепи кругом;
+Идёт направо — песнь заводит,
+Налево — сказку говорит.
+Там чудеса: там леший бродит,
+Русалка на ветвях сидит;
+Там на неведомых дорожках
+Следы невиданных зверей;
+Избушка там на курьих ножках
+Стоит без окон, без дверей;
+Там лес и дол видений полны;
+Там о заре прихлынут волны
+На брег песчаный и пустой,
+И тридцать витязей прекрасных
+Чредой из вод выходят ясных,
+И с ними дядька их морской;
+Там королевич мимоходом
+Пленяет грозного царя;
+Там в облаках перед народом
+Через леса, через моря
+Колдун несёт богатыря;
+В темнице там царевна тужит,
+А бурый волк ей верно служит;
+Там ступа с Бабою Ягой
+Идёт, бредёт сама собой,
+Там царь Кащей над златом чахнет;
+Там русский дух… там Русью пахнет!
+И там я был, и мёд я пил;
+У моря видел дуб зелёный;
+Под ним сидел, и кот учёный
+Свои мне сказки говорил.
+"""
+
+# TODO Распечатайте в столбик букву и её частоту в тексте
+
+
+def count_letters(a):
+    dict_=dict()
+
```

Comment:
Есть еще одна идея. Можно попробовать просто добавить вот такую строчку:
```suggestion
    # Приведение всех символов к нижнему регистру
    a = a.lower()
```

Теперь подсчет должен происходить верно.